app/src/main/java/com/eventyay/organizer/core/event/dashboard/EventDashboardPresenter.java

```diff
@@ -112,7 +112,7 @@ public void confirmToggle() {
         if (Event.STATE_PUBLISHED.equals(event.state)) {
             getView().switchEventState();
             getView().showEventUnpublishDialog();
-        } else if (Utils.isEmpty(event.getLocationName())) {
+        } else if (Utils.isEmpty(event.getLocationName()) && !event.isEventOnline) {
```

Comment:
@sridharjajoo I think the it should be placed first. If there's no location name, the condition `!event.isEventOnline` will not be checked, but an online event can have a location in the case when an event is made online later.